otter/cloud_client.py

```diff
@@ -394,15 +394,20 @@ class CLBImmutableError(Exception):
 
 
 @attributes([Attribute('lb_id', instance_of=six.text_type)])
-class CLBDeletedError(Exception):
+class CLBNotFoundError(Exception):
+    """A CLB doesn't exist. Superclass of other, more specific exceptions."""
+    message = attr.ib(default='')
```

Comment:
This is `attr`-specific attribute declaration, but the decorator is `characteristic`-specific, and already declares the `lb_id` type